pkg/lifecycle/service.go

```diff
@@ -220,7 +220,12 @@ func (s *Service) StartWithBackoff(ctx context.Context, rp *runnablePipeline) er
 		Int64(log.AttemptField, attempt).
 		Msg("restarting with backoff")
 
-	time.AfterFunc(duration+s.errRecoveryCfg.HealthyAfter, func() {
+	time.AfterFunc(duration+s.errRecoveryCfg.MaxRetriesWindow, func() {
+		s.logger.Info(ctx).
```

Comment:
Maybe this should be a debug log?

Reply:
@lovromazgon I questioned myself about it too, so I'll change it to debug.